### ISSUE TITLE
Support DVD ISO mount?

### ISSUE THREAD

**Issue by okudayukiko** [sarah-walker-pcem/pcem#57]
**Describe the bug**
PCem doesn't support DVD ISO mount. e.g. windows11-eng.iso and ubuntu-20.04.2-desktop-amd64.iso.

**Comment by unreal9010** [comment]
I think this issue should be closed.

**Comment by Vaporeon** [comment]
OP's use case might be invalid but I think DVD emulation alone is useful. Late 9x boxes supported them after all.

**Comment by michael-manley** [comment]
Yeah I saw now that it meant its unable to read dvd isos in general. I marked it correctly now.
